### DISCUSSION TITLE
Mouse issues

### DISCUSSION THREAD

**Discussion by jbuchner07** [AntiMicroX/antimicrox#528] [Q&A]
I have been trying to get the joystick to move the mouse to play some games. It is registering the joystick movement, but the mouse will not move. I have the joystick configured to the mouse movements.  Is there something i should be looking at that will allow the mouse movement? I apologize if i missed it in advanced.
![blah](https://user-images.githubusercontent.com/12581133/187124981-fb7a2652-f7f7-4421-84d1-84194a1dc532.JPG)
![blah 2](https://user-images.githubusercontent.com/12581133/187124984-4a6eec85-0f25-4a9e-ad2f-c838736080b5.JPG)
![blah 3](https://user-images.githubusercontent.com/12581133/187124988-3a1cbe59-a05a-40db-a550-8a6a4d8d4fe1.JPG)

**Comment by jbuchner07** [comment]
If this helps anymore, yesterday I did change anything but the controller was moving the mouse. But tonight it is no longer moving, I still haven't touched anything different.

**Reply by pktiuk** [reply]
I don't know what may be wrong.  
Could you describe your setup?  OS, and paste informations from `Help->About->Info` window?
